src/libs/EmojiTrieNode.js

```diff
@@ -0,0 +1,44 @@
+/** Class representing a Trie node. */
+class TrieNode {
```

Comment:
In my opinion we should decouple emojis from our `Trie` implementation. We should implement (and ideally write unit tests for) a generic `Trie` in `src/libs/Trie` and then utilize that data structure for our emoji.